src/main/scala/viper/silver/plugin/standard/inline/InlineRewrite.scala

```diff
@@ -51,6 +59,65 @@ trait InlineRewrite extends PredicateExpansion {
       .execute[Exp](expr)
   }
 
+  /**
+    * Given an inhale statement, look within it for any predicate accesses. If found, return the same inhale statement
+    * with the expanded value of the predicate.
+    *
+    * @param stmt The statement containing the inhale statement we wish to expana.
+    * @param method The method containing the inhale statement we wish to expand.
+    * @param program The Viper program for which we perform this expansion.
+    * @param cond The condition a predicate must satisfy to be expanded within an inhale statement.
+    * @return The expanded inhale statement.
+    */
+  private[this] def expandInhaleStatement(stmt: Stmt, method: Method, program: Program, cond: String => Boolean): Stmt =
+    ViperStrategy.Context[Set[String]]({
+      case tup @ (Inhale(expr), ctxt) =>
+        val expandedExpr = expandInhaleContainedExpr(expr, method, program, cond)
+        tup match {
+          case (inhaleStmt: Inhale, _) =>
+            val expandedInhaleStmt =
+              inhaleStmt.copy(exp = expandedExpr)(pos = inhaleStmt.pos, info = inhaleStmt.info, errT = inhaleStmt.errT)
+            (expandedInhaleStmt, ctxt)
+          case _ =>
+            // This is required to silence the compiler warning for non-exhaustive pattern matching
+            throw new Error("This error should never be thrown")
+        }
```

Comment:
All we need to do to enable support for `Exhale` is to duplicate this block and replace `Inhale` with `Exhale`

I love ADTs